src/numtracker.rs

```diff
@@ -142,10 +142,7 @@ impl GdaNumTracker<'_, '_> {
         if self.ext != file.extension()?.to_str()? {
             return None;
         }
-        match file.file_stem()?.to_str()?.parse() {
-            Ok(val) => Some(val),
-            Err(_) => None,
-        }
```

Comment:
Could this be a separate PR?

Reply:
If you want

Reply:
Done https://github.com/DiamondLightSource/numtracker/pull/79